terracumber_config/tf_files/SUSEManager-Test-Hexagon-NUE.tf

```diff
@@ -179,27 +179,66 @@ module "cucumber_testsuite" {
         // Since start of May we have problems with the instance not booting after a restart if there is only a CPU and only 1024Mb for RAM
         // Still researching, but it will do it for now
         memory = 2048
-        vcpu = 2
       }
       additional_packages = [ "venv-salt-minion" ]
       install_salt_bundle = true
     }
-    pxeboot-minion = {
-      image = "sles15sp4o"
+    debian-minion = {
```

Comment:
in the other files we leave empty lines between host declarations... follow that convention here too?